WooCommerce/Classes/ServiceLocator/ProductImageUploader.swift

```diff
@@ -218,24 +215,32 @@ private extension ProductImageUploader {
     /// Called to replace the local product ID with remote product ID for the previously uploaded images
     ///
     func updateProductIDOfImagesUploadedUsingLocalProductID(siteID: Int64,
-                                                            productID: Int64,
+                                                            productOrVariationID: ProductOrVariationID,
                                                             images: [ProductImage]) {
+        let imageProductOrVariationID: Int64 = {
```

Comment:
Can we add a computed property to `enum ProductOrVariationID` for extracting the ID?

Reply:
good idea! updated in https://github.com/woocommerce/woocommerce-ios/pull/7179/commits/85e3bd621fd194e90e5bb1b222839bf09c99c7a9